packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -409,6 +409,13 @@ class MultiSelectComboBox extends ResizeMixin(InputControlMixin(ThemableMixin(El
         type: Array,
         value: () => [],
       },
+
+      /** @private */
+      _focusedChipIndex: {
```

Comment:
Should the focused chip be announced? Currently, it may come as a surprise for screen reader users that it takes two backspace presses to get the last selected value removed.

Reply:
Good point. I think we should announce it, maybe something like `{chip} focused. Press Backspace to remove`

Reply:
Added `i18n.focused` and used it to announce focusing a chip with keyboard.